guest-os-info.md

```diff
@@ -0,0 +1,49 @@
+## Guest Operating System Information
+
+Guest operating system identity for the Virtual Machine will be provided by the label ``os-short-id`` :
+
+```
+metadata:
+  name: myvm
+  labels:
+    os-short-id: win2k12r2
+```
+
+The ``os-short-id`` label is based on the short OS identifier from [libosinfo](https://libosinfo.org/).
+Libosinfo database can be queried to extract additional information about the operating system using the short-id:
+
+```
+# List all operating systems
+$ osinfo-query os
+
+ Short ID             | Name                                              | Version | ID
+----------------------------------------------------------------------------------------------------------------------
+...
+ fedora24             | Fedora 24                                      | 24        | http://fedoraproject.org/fedora/24
+ fedora25             | Fedora 25                                      | 25        | http://fedoraproject.org/fedora/25
+ fedora26             | Fedora 26                                      | 26        | http://fedoraproject.org/fedora/26
+...
+ win2k12r2          | Microsoft Windows Server 2012 R2 | 6.3       | http://microsoft.com/win/2k12r2
+...
```

Comment:
If you want to have a table with short IDs please make sure it is complete

Reply:
The table here is a shorten version of the ``osinfo-query os`` output (that include all short IDs)
I didnt see a need to show the whole table here

Reply:
In my opinion if you create something partial like above you either finalize it or give a reference where one can find complete list of `short-id`s.